README.md

```diff
@@ -54,6 +54,32 @@ Starting from zone.js v0.8.9, you can choose which web API module you want to pa
 For more details, please
 see [MODULE.md](MODULE.md).
 
+## Bundles
+There are several bundles under `dist` folder.
+- `zone.js`, the default bundle, contains the most used APIs such as `setTimeout/Promise/EventTarget...`, also this bundle will support all evergreen and legacy (IE/Legacy Firefox/Legacy Safari) Browsers
```

Comment:
Use common tense throughout please. "will support" -> supports

Reply:
Can you also explicitly say that zone.js = zone-evergreen.js + zone-legacy.js

Reply:
Sure, got it.